Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -4,10 +4,12 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
 using Unity.Profiling;
 using UnityEngine;
 using UnityEngine.EventSystems;
 
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.SDK")]
```

Comment:
We may want to throw this into an AssemblyInfo.cs for discoverability. You'll likely need to create one for this assembly, but the pipeline patching step should adjust.

Reply:
Updated!